packages/react-dom/src/client/ReactDOMComponent.js

```diff
@@ -8,7 +8,7 @@
  */
 
 import {
-  registrationNameModules,
+  registrationNames,
```

Comment:
This used to be an object hash of name -> plugin. But the plugin object wasn't actually used. I changed it to a hash of name -> true. This lets us get rid of object "plugin" representations.